class.jetpack-jitm.php

```diff
@@ -272,6 +305,43 @@ function editor_msg() {
 		}
 	}
 
+	/**
+	 * Display message in editor prompting user to enable stats.
+	 *
+	 * @since 3.9
+	 */
+	function stats_msg() {
+		if ( current_user_can( 'manage_options' ) ) {
```

Comment:
is this necessary given we check `current_user_can( 'jetpack_manage_modules' )` in `prepare_jitms()`?